src/cms/templates/pages/page_tree_node.html

```diff
@@ -33,6 +33,13 @@
             {{ page_translation.get_status_display }}
         </a>
     </td>
+    <td>
```

Comment:
```suggestion
    <td class="whitespace-nowrap">
```
Adding the class `whitespace-nowrap` to the whole table cell makes sure that the labels are not split into multiple lines.